plugins/module_utils/foreman_helper.py

```diff
@@ -54,7 +54,7 @@ def parse_params(self):
         return {k: v for (k, v) in self._params.items() if v is not None}
 
     def clean_params(self):
-        return {k: v for (k, v) in self._params.items() if v is not None}
+        return {k: v for (k, v) in self._params.items() if v is not None and k is not 'verify_ssl'}
```

Comment:
Can we clean any alias and only keep the "original" name?

Reply:
I don't know how easy that would be, and i'd expect ansible to do this already.